theano/configdefaults.py

```diff
@@ -640,6 +640,14 @@ def warn_default(version):
     in_c_key=False)
 
 
+AddConfigVar(
+    'print_test_value_by_default',
```

Comment:
I would name it print_test_value

It is shorter and to me, it is as much informative. I currently don't see possible futur colision of flag name with that. If you see some, tell me. We can discuss this.